src/main/java/no/nav/familie/integrasjoner/arbeidsfordeling/ArbeidsfordelingService.kt

```diff
@@ -54,5 +62,42 @@ class ArbeidsfordelingService(
         }
     }
 
+    fun finnBehandlendeEnhetForPersonMedRelasjoner(personIdent: String, tema: Tema): List<Enhet> {
```

Comment:
Dette gjelder ju den eldre versjonen og, men en liste med enhet med kun 2 felter `enhetId` og `enhetId`, hvordan skal vi vite hvilken vi skal bruke? :P Finnes det flere felter i Enhet som vi burde forholde oss til?

Reply:
Jeg knotet MYE med dette enhetstullet i forrige uke, men det gikk opp et lys da jeg fant et api som var "beste-match": den gir en listen med 1 eller 0 tilbake - så da kan vi fortsette å gjøre som før: `.firstOrNull()`

De gamle gir også en slags "beste-match". I EF kunne vi laget disse reglene enkelt ( i praksis 3 enheter - kode6, egen ansatt og resten), men for BAR, OPP og andre temaer så er det ikke like trivielt hvilken enhet personen skal lande på. Og nei, det er ikke så veldig mange felter å "filtrere" på selv.